.github/workflows/node.yml

```diff
@@ -26,4 +26,5 @@ jobs:
       - run: npm run lint
       - run: npm run build
       - run: npm test
+      - run: GROUP_CONS=noble npm test
```

Comment:
no changes needed if we run variants in Jest.

Reply:
Yeah, I think defaulting to running all the tests in jest makes sense, though equally, a way to test only a single group without having to edit some file, also does.